_data/people/masonproffitt.yml

```
@@ -57,3 +57,11 @@ presentations:
   location: Virtual
   focus-area: as
   project: func-adl
+- title: "FuncADL: Functional Analysis Description Language"
+  date: 2021-07-09
+  url: https://github.com/masonproffitt/pyhep-2021_func-adl/blob/pyhep_2021_final/FuncADL%20at%20PyHEP%202021.ipynb
```

Could you link to the actual indico entry, not the notebook? The indico link is more useful and also has this link, and the URL is stable.

---

```suggestion
  url: https://indico.cern.ch/event/1019958/timetable/#31-funcadl-functional-analysis
```

this is the Indico link to Mason's talk.

---

```suggestion
  url: https://indico.cern.ch/event/1019958/timetable/#31-funcadl-functional-analysis
  video: https://youtu.be/58KR97xdYlU
```

Can get the video recording linked in there too. :)